k8sfv/k8sfv.go

```diff
@@ -158,7 +158,7 @@ func initialize(k8sServerEndpoint string) (clientset *kubernetes.Clientset) {
 
 func initializeCalicoDeployment(k8sServerEndpoint string) {
```

Comment:
why cant we just get rid of this entire function? We don't need any initialization anymore, and the client created is not used (as it's `_`'d out)

Reply:
yep, good suggestion, will do.